src/dotnet/Chat.UI.Blazor/Components/Onboarding/AvatarStep.razor

```diff
@@ -144,7 +146,6 @@
     }
 
     private void OnImagePicked(MediaContent mediaContent) {
-        _stepModel.Picture = mediaContent.ContentId;
```

Comment:
if OnGenerateAvatarClick was executed Picture has a link to a picture
seems like set it to "" will be useful